src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -482,14 +482,14 @@ private void ValidateEnumDependencies(EnumDependenciesSchema enumDependencies, s
             return (name, 0);
         }
 
-        private Error OverwriteError(JsonSchema schema, string name, Error baseError)
+        private Error GetAdditionalError(JsonSchema schema, string name, Error baseError)
         {
-            if (!string.IsNullOrEmpty(name) && schema.OverwriteErrors.TryGetValue(name, out var attributeOverwriteErrors) && attributeOverwriteErrors.TryGetValue(baseError.Code, out var overwriteError))
+            if (!string.IsNullOrEmpty(name) && schema.AdditionalErrors.TryGetValue(name, out var attributeAdditionalErrors) && attributeAdditionalErrors.TryGetValue(baseError.Code, out var additionalError))
             {
                 return new Error(
-                    !overwriteError.Severity.HasValue ? baseError.Level : overwriteError.Severity.Value,
-                    string.IsNullOrEmpty(overwriteError.Code) ? baseError.Code : overwriteError.Code,
-                    string.IsNullOrEmpty(overwriteError.Message) ? baseError.Message : overwriteError.Message,
+                    !additionalError.Severity.HasValue ? baseError.Level : additionalError.Severity.Value,
+                    string.IsNullOrEmpty(additionalError.Code) ? baseError.Code : additionalError.Code,
+                    string.IsNullOrEmpty(additionalError.Message) ? baseError.Message : $"{baseError.Message}. {additionalError.Message}",
```

Comment:
Some of our error message ends with period. can `Debug.Assert` error message ends with '.' in Errors.cs line 40 and drop the `.` here

Reply:
sure, i will fix it